website/docs/cluster-management/deploying-capa-eks.mdx

```diff
@@ -0,0 +1,214 @@
+
+import WegoAdmin from "!!raw-loader!./assets/rbac/wego-admin.yaml";
+
+<BrowserOnly>
+  {() => (
+    <CodeBlock className="language-bash">
+      curl -o clusters/bases/rbac/wego-admin.yaml {window.location.protocol}//
+      {window.location.host}
+      {require("./assets/rbac/wego-admin.yaml").default}
+    </CodeBlock>
+  )}
+</BrowserOnly>
+
+<details><summary>Expand to see full template yaml</summary>
+
+<CodeBlock
+  title="clusters/bases/rbac/wego-admin.yaml"
+  className="language-yaml"
+>
+  {WegoAdmin}
+</CodeBlock>
+
+</details>
+
+## 3. Build a Kubernetes Platform with Built-in Components Preconfigured for Your Organization
+
+To do this, go to Weaveworks' [Profiles Catalog](https://github.com/weaveworks/profiles-catalog).
+
+See [CAPI Templates](gitops-templates/intro.mdx) page for more details on this topic. Once we load a template we can use it in the UI to create clusters!
+
+import CapaTemplate from "!!raw-loader!./assets/templates/capa-template.yaml";
+
+Download the template below to your config repository path, then commit and push to your Git origin.
+
+<BrowserOnly>
+  {() => (
+    <CodeBlock className="language-bash">
+      curl -o clusters/management/capi/templates/capa-template.yaml{" "}
+      {window.location.protocol}//{window.location.host}
+      {require("./assets/templates/capa-template.yaml").default}
+    </CodeBlock>
+  )}
+</BrowserOnly>
+
+<CodeBlock
+  title="clusters/management/apps/capi/templates/capa-template.yaml"
+  className="language-yaml"
+>
+  {CapaTemplate}
+</CodeBlock>
+
+## 4. Add a Cluster Bootstrap Config
+
+This step ensures that Flux gets installed into your cluster. Create a cluster bootstrap config as follows:
+
+```bash
+ kubectl create secret generic my-pat --from-literal GITHUB_TOKEN=$GITHUB_TOKEN
+```
+
+import CapiGitopsCDC from "!!raw-loader!./assets/bootstrap/capi-gitops-cluster-bootstrap-config.yaml";
+
+Download the config with:
+
+<BrowserOnly>
+  {() => (
+    <CodeBlock className="language-bash">
+      curl -o
+      clusters/management/capi/bootstrap/capi-gitops-cluster-bootstrap-config.yaml{" "}
+      {window.location.protocol}
+      //{window.location.host}
+      {
+        require("./assets/bootstrap/capi-gitops-cluster-bootstrap-config.yaml")
+          .default
+      }
+    </CodeBlock>
+  )}
+</BrowserOnly>
+
+Then update the `GITOPS_REPO` variable to point to your cluster
+
+<details><summary>Expand to see full yaml</summary>
+
+<CodeBlock
+  title="clusters/management/capi/boostrap/capi-gitops-cluster-bootstrap-config.yaml"
+  className="language-yaml"
+>
+  {CapiGitopsCDC}
+</CodeBlock>
+
+</details>
+
+## 5. Delete a Cluster with the Weave GitOps Enterprise UI
+
+Here are the steps:
+- Select the clusters you want to delete
+- Press the `CREATE A PR TO DELETE CLUSTERS` button
+- Either update the deletion PR values or leave the default values, depending on your situation
+- Presstje `Remove clusters` button
```

Comment:
```suggestion
- Press `Remove clusters` button
```

Reply:
Maybe a screenshot?

Also, are some buttons really _ALL UPPER CASE_ and others not?